tests/test_middleware.py

```diff
@@ -47,7 +47,7 @@ def test_wrong_default_hosts(self):
         ROOT_HOSTCONF='tests.hosts.simple',
         DEFAULT_HOST='www')
     def test_request_urlconf_module(self):
-        rf = RequestFactory(HTTP_HOST='other.example.com')
+        rf = RequestFactory(headers={"host": 'other.example.com'})
```

Comment:
Would be great if you could match the quotation marks with the existing ones 🙂
```suggestion
        rf = RequestFactory(headers={'host': 'other.example.com'})
```
(I think we should start using Black or the Ruff formatter, which make all quotation marks double (by default), but that's for another time :))

Reply:
@mikemanger I guess I should have stated explicitly that this comment also applies to the other instances of `"host"` in this file and `test_sites.py` :p

Reply:
Nvm, I just pushed a commit for it 🙂